### ISSUE TITLE
Not able to edit a doc file

### ISSUE THREAD

**Issue by joydipnath** [PHPOffice/PHPWord#1220]
This is my code. I just want to open an existing docx file and then write some text at the beginning of the docx file.

```php
$source = storage_path('helloWorld.docx');
$phpWord = \PhpOffice\PhpWord\IOFactory::load($source, 'Word2007');
$section = $phpWord->addSection();
$header = array('size' => 16, 'bold' => true);
$section->addText('Hello 1', $header);
$objWriter = \PhpOffice\PhpWord\IOFactory::createWriter($phpWord, 'Word2007');
$objWriter->save(storage_path('helloWorld.docx'));
```

It's like very basic operation for editing itself is showing error, What did is missed?  
Any help would be appreciated.

**Comment by troosan** [comment]
@joydipnath what is the error you are getting?

**Comment by troosan** [comment]
I modified the Sample_11_ReadWord2007.php as follows and it works fine.

```php
<?php
include_once 'Sample_Header.php';

// Read contents
$name = basename(__FILE__, '.php');
$source = __DIR__ . "/resources/{$name}.docx";

echo date('H:i:s'), " Reading contents from `{$source}`", EOL;
$phpWord = \PhpOffice\PhpWord\IOFactory::load($source, 'Word2007');
$section = $phpWord->addSection();
$header = array('size' => 16, 'bold' => true);
$section->addText('Hello 1', $header);

// Save file
echo write($phpWord, basename(__FILE__, '.php'), $writers);
if (!CLI) {
    include_once 'Sample_Footer.php';
}
```

**Comment by joydipnath** [comment]
This is the error I am getting
`InvalidArgumentException
Invalid style value: none Options:bullet,decimal,upperRoman,lowerRoman,upperLetter,lowerLetter`

**Comment by troosan** [comment]
this was fixed in the current development branch.
Can you try with version dev-develop?

**Comment by troosan** [comment]
fixed in 0.14, will be release 28/12
